src/AstrometryFit.cc

```diff
@@ -717,9 +717,9 @@ unsigned AstrometryFit::findOutliers(double nSigCut, MeasuredStarList &msOutlier
 void AstrometryFit::removeMeasOutliers(MeasuredStarList &outliers) {
     for (auto &i : outliers) {
         MeasuredStar &ms = *i;
-        auto fs = std::const_pointer_cast<FittedStar>(ms.GetFittedStar());
-        ms.SetValid(false);
-        fs->MeasurementCount()--;  // could be put in SetValid
+        auto fs = std::const_pointer_cast<FittedStar>(ms.getFittedStar());
+        ms.setValid(false);
+        fs->getMeasurementCount()--;  // could be put in setValid
```

Comment:
Because I'm not sure I believe the comment, but I'm also not sure I don't (which is why I haven't deleted it).